etc/listproviders/event.sources.properties

```diff
@@ -0,0 +1,5 @@
+list.name=eventSourcesFilter
+list.translatable=false
```

Comment:
Thanks for factoring out the hard-coded lists, but as for this list, I don't think that it is of any use. Please remove the list "eventSourcesFilter" or provide a reason why you think it is needed.

Reply:
You're right, these lists are no longer in use, I overlooked that. Will remove them.